app/controllers/hyrax/batch_ingest/batches_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Hyrax
   module BatchIngest
-    class BatchesController < Hyrax::BatchIngest::ApplicationController
+    class BatchesController < Hyrax::MyController
```

Comment:
This seems fine to me but maybe it would be better to leave `BatchesController` subclassing `Hyrax::BatchIngest::ApplicationController` and change `Hyrax::BatchIngest::ApplicationController` to subclass `Hyrax::MyController` instead?

Reply:
sure, I can do that. Although I'm not sure what are _all_ the behaviors that come in from `Hyrax::MyController`. I assume we don't need everything. But we can roll with what we have for now, and pick it apart later if necessary.